src/routes/routes.ts

```diff
@@ -0,0 +1,7 @@
+import { Router } from "express";
+
+const router: Router = Router();
+
+router.get("/", (req, response) => response.send("This is the Strike Off Objections app &#128516;"));
```

Comment:
What's the magic incantation at the end of the line? Perhaps describe in a comment.

Reply:
you'll have to run it and see

Reply:
it's gone now :(